src/daos/section-dao.js

```diff
@@ -1,14 +1,68 @@
-import sections from '../mock-data/raw/sections-raw'
 
-export function getSection(sectionID) {
-  return new Promise((resolve, reject) => {
-    if (typeof sectionID === 'string' && !isNaN(sectionID))
-      sectionID = parseInt(sectionID, 10)
-    const res = sections.filter(section => section.sectionID === sectionID)
-    if (res.length === 1) {
-      resolve(res[0])
-    } else {
-      reject(new Error(`Unable to find section with ID: ${sectionID}`))
+export function getSection(sectionID, db) {
+    return new Promise((resolve, reject) => {
+        db.query('select *, student.firstName as studentFName, student.lastName as studentLName ' +
+            'from student natural join ydsd natural join takes natural join section natural join teaches, staff where sectionID = ? and staff.emailID = teaches.emailID',
+            sectionID, function (err, entities) {
+                if (err) {
+                    reject(err)
+                } else {
+                    var result = {
```

Comment:
It has to do with scoping. You don't really ever use `var` in ES6. See [this](http://softwareengineering.stackexchange.com/questions/274342/is-there-any-reason-to-use-the-var-keyword-in-es6)